### PR TITLE
makeup

### PR DESCRIPTION
**Pull request by Livitan**
bit layout

### CODE REVIEW

**Comment by firec0der** [comment]
Could you please attach some screenshots?

**Comment by Livitan** [comment]
desktop
mobile

File: _data/streams.yml [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+- date: "25 May 2016"
+  datetime: "2016-05-25"
+  topic: "Apollo/GraphQL/Relay/Falcor"
+  url: "https://www.youtube.com/embed/C3OqbN-Rtro"
+- date: "25 May 2016"
+  datetime: "2016-05-25"
+  topic: "Apollo/GraphQL/Relay/Falcor"
+  url: "https://www.youtube.com/embed/C3OqbN-Rtro"
+- date: "25 May 2016"
+  datetime: "2016-05-25"
```
Comment (firec0der) [inline _data/streams.yml]: I think that you forgot to remove unnecessary items :)

File: _includes/header.html [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-  <h2>Next session</h2>
-  <p>
-    Date: {{ header.date }}
-    Topic: {{ header.topic }}
-    {% unless header.url == '' %}
-      Stream: {{ header.url }}
-    {% endunless %}
-  </p>
+<section class="announcements">
+  <h2>Next meeting on {{ header.date }}</h2>
```
Comment (firec0der) [inline _includes/header.html]: Please, add a topic

File: _includes/streams.html [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+<section class="feed row">
+  {% for stream in streams %}
+  <article class="stream small-12 medium-12 large-6 columns">
+    <h4 class="streams-topic">
+      {{ stream.topic }}
+    </h4>
+    <time class="streams-date" datetime="{{ stream.datetime }}">{{ stream.date }}</time>
+    <iframe width="100%" height="315px" src="{{ stream.url }}" frameborder="0" allowfullscreen></iframe>
+  </article>
+  {% endfor %}
```
Comment (firec0der) [inline _includes/streams.html]: Please, fix the indent in the `<main></main>`

File: assets/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
+body {
+  background-color: cornsilk;
+}
+header {
+text-align: center;
+padding-top: 2em;
+}
```
Comment (firec0der) [inline assets/css/styles.css:7]: Here is also problem with the indent.

File: assets/css/vendor/normalize.css [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+td,
+th {
+  padding: 0;
+}
```
Comment (firec0der) [inline assets/css/vendor/normalize.css]: Please, add the minified version of `normalize`

File: assets/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+header p {
+  margin-bottom: 0.2em;
+}
+.announcements {
+  margin-top: 3em;
+}
+.announcements h2 {
```
Comment (firec0der) [inline assets/css/styles.css]: `.announcements > h2`?

File: _includes/footer.html [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{% assign footer = site.data.footer %}
+
+<footer class="page-footer">
+  <p>made with ❤︎</p>
```
Comment (firec0der) [inline _includes/footer.html:4]: 👍

**Comment by Livitan** [comment]
A new version of the fix
desktop

File: _includes/head.html [hunk 8]
```diff
@@ -44,5 +44,7 @@
   <link rel="shortcut icon" type="image/x-icon" href="/favicon.png">
 
   <!-- CSS STYLES -->
+  <link rel="stylesheet" href="/assets/css/vendor/normalize.min.css"/>
+  <link rel="stylesheet" href="/assets/css/vendor/foundation.min.css"/>
```
Comment (sudodoki) [inline _includes/head.html:48]: What styles are used from foundation?

**Comment by sudodoki** [comment]
LGTM as initial version, can improve afterwards.

File: _includes/head.html (hunk 8 above)
Comment (Livitan) [inline _includes/head.html:48]: Only grid and some typography
`<article class="stream small-12 medium-12 large-6 columns">`

**Pull request merged**